### ISSUE TITLE
Use first partition for data so that it can be used on windows

### ISSUE THREAD

**Issue by netixx** [aguslr/multibootusb#217]
Windows only recongnise/mounts the first partition on removable drives.

By using the first partition for data (instead of the second), the drive could be used to transfer ISO downloaded on windows or as a regular flash drive.

**Comment by netixx** [comment]
Yes I know, the idea was to change the default order so that it works out of the box.

This would be useful for beginners and save time for admins. What I don't know is if there is a reason for using the first partition for EFI or caveats when you don't.

**Comment by aguslr** [comment]
No problem there, according to the [UEFI Spec](http://www.uefi.org/sites/default/files/resources/UEFI_Spec_2_7.pdf), section 13.3.3:

> UEFI does not impose a restriction on the number or location of System Partitions that
can exist on a system.

On the other hand, the problem might be the [BIOS Boot partition](https://en.wikipedia.org/wiki/BIOS_boot_partition):

> On MBR disks, such boot loaders typically use the sectors immediately following the MBR for this storage; that space is usually known as the "MBR gap". No equivalent unused space exists on GPT disks, and the BIOS boot partition is a way to officially allocate such space for use by the boot loader.
